how-we-work-together.md

```diff
@@ -1,368 +1,368 @@
-
-This document is meant to capture the typical responsibilities we accept when we are performing three particular disciplines on a project:
-
-* Managing the project
-
-* Designing
-
-* Developing
-
-## When We Manage a Project... ⏱
-
-### We Do Business Development
-
-As we manage a project, we have a unique perspective and opportunity to develop our business *while* we develop the business of our clients. This is a balance, building mutually beneficial, valuable situations between Sparkbox and our customers as we...
-
-* Contribute to proposals and estimates for new business opportunities 
-
-* Coordinate proposals and estimates of new work for existing clients 
-
-* Manage handoff from our outreach team to the project team
-
-* Run discovery (when applicable)
-
-* Coordinate and/or write project briefs (when applicable)
-
-* Promote next project phase during current phase
-
-* Encourage the team to identify more ways Sparkbox can add value
-
-* Write case studies
-
-### We Manage Expectations
-
-No other person has as broad and nuanced a project view as the project's manager (PM). While the client and team may be invested and informed, the PM has the unique opportunity and responsibility to draw from the entire contributing landscape and provide appropriate context for each person involved in the project. We manage expectations so that each person involved in the project is equipped to perform at their best.
-
-* For the team working on the project
-
-    * Listen to the team and establish trust
-
-    * Make sure the team understands the project’s goals, constraints, and client background/expectations 
-
-        * Also communicate Sparkbox’s internal goals for the project (e.g. "this is going to be an important portfolio piece, because...")
-
-    * Understand the work well enough to make informed decisions
-
-    * Correct the team when their expectations don’t align with project goals and client expectations
-
-    * Set the team up for success
-
-        * Do whatever you can to unblock work
-
-        * Protect team members’ time
-
-        * Discuss team members’ time when billable expectations aren’t met
-
-        * Provide team care when needed (order lunch, etc)
-
-* For the client
-
-    * Listen to the client and establish trust
-
-    * Understand their needs
-
-        * Dig to the root of stated needs
-
-        * Identify value we can deliver
-
-        * Continue to dig throughout project
-
-    * Understand the work well enough to know when the client needs to make a decision
-
-    * When the client’s expectations don’t align with the project goals or the reality of the project’s timeline, scope, or budget, the client’s expectations need to be corrected ASAP or the timeline, scope, or budget must change.
-
-    * Help your point of contact look good
-
-* For Sparkbox
-
-    * Provide regular project health updates
-
-    * Expose risks to the ops team
-
-        * How the project impacts other projects
```

Comment:
Changed because the first point is written in present tense, while the other two are written in present progressive. Changed the first to match.